src/Component/Router.purs

```diff
@@ -79,40 +64,37 @@ component
   => ManageComment m
   => ManageTag m
   => H.Component HH.HTML Query Input Void m
-component =
-  H.parentComponent
-    { initialState: \initialRoute -> { route: fromMaybe Home initialRoute } 
-    , render
-    , eval
-    , receiver: const Nothing
-    }
-
+component = H.mkComponent
+  { initialState: \initialRoute -> { route: fromMaybe Home initialRoute } 
+  , render
+  , eval: H.mkEval $ H.defaultEval { handleQuery = handleQuery }
+  }
   where 
+  handleQuery :: forall a. Query a -> H.HalogenM State Void ChildSlots Void m (Maybe a)
+  handleQuery = case _ of
+    Navigate dest a -> do
+      { route } <- H.get 
+      when (route /= dest) do
+        H.modify_ _ { route = dest }
+      pure (Just a)
 
-  eval :: Query ~> H.ParentDSL State Query ChildQuery ChildSlot Void m
-  eval (Navigate dest a) = do
-    { route } <- H.get 
-    when (route /= dest) do
-      H.modify_ _ { route = dest }
-    pure a
-
-  render :: State -> H.ParentHTML Query ChildQuery ChildSlot m
+  render :: State -> H.ComponentHTML Void ChildSlots m
   render { route } = case route of
     Home -> 
-      HH.slot' CP.cp1 unit Home.component unit absurd
+      HH.slot (SProxy :: _ "home") unit Home.component unit absurd
```

Comment:
In Halogen 4 we used a child path and slot type to identify a particular component among many. In Halogen 5 we use a symbol proxy and a slot type instead. 

A symbol proxy is a type-level string. You've seen them before: they're used as labels for row types (and records and variants, which are built on top of rows). In this case we're identifying the component with the label `home` in our `ChildSlots` row defined at the top of this module.

I've written `SProxy :: _ "home"` as a shorthand here, but the full type is `SProxy :: SProxy "home"`. That means that at the value level we have just `SProxy`, but this value has the type `SProxy "home"`, which is how we're able to use the label "home" at the type level.